ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -94,7 +72,7 @@ echo "export MIRROR_IMAGES=true" >> /root/dev-scripts/config_root.sh
 
 echo 'export KUBECONFIG=/root/dev-scripts/ocp/ostest/auth/kubeconfig' >> /root/.bashrc
 
-timeout -s 9 105m make
+#timeout -s 9 105m make > output.log
```

Comment:
Why > output.log? I think this would prevent some things being indexed by OpenShift CI Search. Could we do `| tee` instead?

Reply:
The idea was to minimize as much as possible the security risks. In any case the dev-scripts logs are available as logs for troubleshooting.